DateRangeBox: add code snippets and extended description #5043 (#5111)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxDateRangeBox/1 Configuration/value.md

---
---
##### shortDescription

> An array of date range values currently selected using the date box. 

Reviewer: Looks like this array sets two values - start and end? 

The phrase "array of date range values" implies that each value is a "date range" and you could select multiple ranges. And I don't think that's the case. 

```suggestion
An array that specifies the selected range (start and end dates).
```

Author: Yes, users can specify the start and end dates in this array or use startDate/endDate properties instead.

---

> You can specify the date range with any of the following formats:

Reviewer: ```suggestion
You can specify dates in any of the following formats:
```

- Date  

>  Specifies dates directly.

Reviewer: ```suggestion
 Specifies a date object.
```

- Number  
 Specifies a date as a timestamp (total milliseconds since 1970/01/01).

- String  
 Specifies a date as a string value (`"yyyy-MM-dd"` - "2017-03-06").
 

> When you changed the date value, the new value is saved in the same format as the initial value. We recommend that you specify dates in <a href="https://en.wikipedia.org/wiki/ISO_8601" target="_blank">UTC ISO 8601</a>.

Reviewer: ```suggestion
If you change a date value, the new value is saved in the same format as the initial value. We recommend that you specify dates in <a href="https://en.wikipedia.org/wiki/ISO_8601" target="_blank">UTC ISO 8601</a>.
```
